src/js/gallery.js

```diff
@@ -0,0 +1,43 @@
+import { fetchTheMovieDBList } from './api'
+import { dataMovies } from "./home";
+
+dataMovies = {
+       page: 1, // nr of actual page
+       totalPages: 20, // nr of total pages
+       query: "", // searching text in home
+     };
+
+     await fetchTheMovieDBList(1, "avatar"); 
+
+// const movies = async () => {
+//     const movies = await fetchTheMovieDBList(2);
+//     console.log(movies) //list of movies
+//     ... <!-- use result -->
+// }
```

Comment:
zadeklaruj zmienną elementu galerii

utwórz export funkcji z parametrem movies np. createMovies(movies) {...}
w funkcji wstaw `const markup = movies.data.map(data => )` i uzupełnij tym co masz poniżej w markup
następnie w funkcji daj `libraryGrid.innerHTML = markup`

i t wszystko. W home.js zamień createMovies() na createMovies(movies). I galeria gotowa

jak zrobisz funkcję to resztę poniżej zakomentuj